src/action/setting.js

```diff
@@ -0,0 +1,22 @@
+import { UNITS } from '../config';
+
+class SettingAction {
+  constructor(store) {
+    this._store = store;
+  }
+
+  //
+  // Bitcoin Unit actions
+  //
+
+  setBitcoinUnit(unit) {
+    if (this._store.settings.unit === unit) {
+      return;
```

Comment:
This check shouldn't really be necessary. No harm done in setting the same value again or am I missing something?